apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -179,6 +179,18 @@ export default class EnrollmentsPanel extends React.Component {
     });
   };
 
+  getViewSurveyUrl = (workshopId, course, subject) => {
+    if (
+      !['CS Discoveries', 'CS Principles', 'CS Fundamentals'].includes(course)
+    ) {
+      return null;
+    }
+
+    return course === 'CS Fundamentals' && subject === 'Intro'
```

Comment:
@molly-moen For CSF Intro, when are we going to switch over to the new survey?

Reply:
our current target is mid-May